R/sensors.R

```diff
@@ -27,6 +27,79 @@ sensor_features <- function(sensor_data, transform, extract, extract_on, groups)
   return(features)
 }
 
+#' Extract tapping (screen sensor) features
+#' 
+#' @param tap_data A data frame with columns t, x, y, buttonid containing 
+#' tapping measurements. buttonid can be from c('TappedButtonLeft','TappedButtonRight','TappedButtonNone') 
+#' indicating a tap that has been classified as to the left, right or neither of those places on the screen
+#' @param depressThr A numerical threshold for intertap distance in x axis
+#' @return A dataframe of features.
+tapping_features <- function(tap_data,
```

Comment:
Move immediately before or after `accelerometer_features` and `gyroscope_features`.